aspnet/SameSite/system-web-samesite.md

```diff
@@ -10,20 +10,59 @@ uid: samesite/system-web-samesite
+Applications that use `iframe` may experience issues with `SameSite=Lax` or `SameSite=Strict` cookies because iframes are treated as
+cross-site scenarios.
 
-Each ASP.NET component that emits cookies needs to decide if SameSite is appropriate.
+The `SameSite=None` option may cause compatibility problems with clients that implemented the prior [2016 draft standard](https://tools.ietf.org/html/draft-west-first-party-cookies-07) (for example, iOS 12). See [Supporting older browsers](#sob) in this document.
 
-## API usage with SameSite
+Each ASP.NET component that emits cookies needs to decide if SameSite is appropriate.
 
-See [HttpCookie.SameSite Property](/dotnet/api/system.web.httpcookie.samesite#System_Web_HttpCookie_SameSite)
+Please see [Known Issues](#known) for problems with applications after installing the 2019 .Net SameSite updates.
+
+## Using SameSite in ASP.NET 4.7.2 and 4.8
+
+.Net currently supports the [2019 draft standard](https://tools.ietf.org/html/draft-west-cookie-incrementalism-00) for SameSite.
+Developers are able to programatically control the value of the SameSite header using the [HttpCookie.SameSite Property](/dotnet/api/system.web.httpcookie.samesite#System_Web_HttpCookie_SameSite). Setting this property to `Strict`, `Lax`, or `None`
+will result in those values being written on the network with the cookie. Setting it equal to `(SameSiteMode)(-1)` indicates that
+no SameSite header should be included on the network with the cookie. The [HttpCookie.Secure Property](/dotnet/api/system.web.httpcookie.secure) (or 'requireSSL' in config files) can be used to mark the cookie as `Secure` or not.
+
+New `HttpCookie` instances will default to `SameSite=(SameSiteMode)(-1)` and `Secure=false`. These defaults can be overridden in
+the `system.web/httpCookies` configuration section, where the string "Unspecified" is a friendly configuration-only syntax for `(SameSiteMode)(-1)`.
+```xml
+<configuration>
+ <system.web>
+  <httpCookies sameSite="[Strict|Lax|None|Unspecified]" requireSSL="[true|false]" />
+ <system.web>
+<configuration>
+```
+
+ASP.Net also issues four specific cookies of its own for these features: Anonymous Authentication, Forms Authentication, Session State,
+and Role Management. Instances of these cookies obtained in runtime can be manipulated using the `SameSite` and `Secure` properties
+just like any other HttpCookie instance. However, due to the patchwork emergence of the SameSite standard, configuration options for
+these four features cookies is inconsistent. The relevant configuration sections and attributes (with defaults) are below. If a
+SameSite/Secure-related attribute exists for a feature, then the feature will use it's own default or configured value. If there is
```

Comment:
This sounds like it needs a table listing each cookie, its default, and where that can be changed.